ntbs-ui-tests/Features/CreateNotifications.feature

```diff
@@ -95,6 +97,26 @@ Feature: Notification creation
     And I can see the value 'Dr Lawton' for the field 'consultant' in the 'HospitalDetails' overview section
     And I can see the value '01 Jan 2019' for the field 'notification-date' in the 'PatientDetails' overview section
 
+  @CookieOverride

```

Comment:
This introduces quite a lot of duplication, just for the fact that we don't want to set (and then look for in the Overview) the user. TDid you look at making a new step to use for the Case Manager input instead which checked the config value for CookieOverride or NormalAuth and ran the step conditionally?

Reply:
I initially avoided doing this because I didn't want to have a step that says 'And I enter x as the case manager' / 'And I can see x as the case manager', but then you look at the step and see it's not actually doing that. I guess it's a trade off either way, and maybe avoiding duplication is worth it (plus it's a pattern we're already using with the 'Given I am logged in as x user' step which is overridden on UAT, and just logs in as you). On reflection, I think you're right!